.gitignore

```diff
@@ -1,5 +1,5 @@
-# Created by https://www.gitignore.io/api/vim,venv,pydev,flask,dotenv,django,direnv,python,virtualenv,pycharm+all,visualstudio,jupyternotebooks
-# Edit at https://www.gitignore.io/?templates=vim,venv,pydev,flask,dotenv,django,direnv,python,virtualenv,pycharm+all,visualstudio,jupyternotebooks
+# Created by https://www.toptal.com/developers/gitignore/api/vim,venv,pydev,flask,dotenv,django,direnv,python,virtualenv,pycharm+all,visualstudio,jupyternotebooks,macos,linux,windows
+# Edit at https://www.toptal.com/developers/gitignore?templates=vim,venv,pydev,flask,dotenv,django,direnv,python,virtualenv,pycharm+all,visualstudio,jupyternotebooks,macos,linux,windows
```

Comment:
```suggestion
# Created by https://www.toptal.com/developers/gitignore/api/vim,venv,pydev,flask,dotenv,django,direnv,python,virtualenv,pycharm+all,visualstudio,visualstudiocode,jupyternotebooks,macos,linux,windows
# Edit at https://www.toptal.com/developers/gitignore?templates=vim,venv,pydev,flask,dotenv,django,direnv,python,virtualenv,pycharm+all,visualstudio,visualstudiocode,jupyternotebooks,macos,linux,windows
```

This is needed otherwise `.vscode/` folder is not ignored.

Adding `VisualStudioCode` to the list, will add the following to the `.gitignore` file. I suggest we comment out the exceptions (starting with `!`) (as I did below) since not everyone in the team/organisation uses vs code, and if others are using vs code, we risk saving and sharing settings that are custom to our local environment messing up with theirs. Instead of commenting out, we could just delete those lines - up to you.

```
### VisualStudioCode ###
.vscode/*
#!.vscode/settings.json
#!.vscode/tasks.json
#!.vscode/launch.json
#!.vscode/extensions.json

### VisualStudioCode Patch ###
# Ignore all local history of files
**/.history

```

Reply:
Ah good shout! Made the change in [`89970d8`](https://github.com/ukgovdatascience/govcookiecutter/pull/12/commits/89970d8f007a4df08a82f53d26dafd77ecc1d3d0).